### ISSUE TITLE
Deleting a Taxonomy Term that is used for a Collection Grid Type value results in a fatal error when the Collection Grid is viewed.

### ISSUE THREAD

**Issue by jwfuller** [CuBoulder/express_mono#685]
![Screen Shot 2020-08-31 at 8.13.09 AM.png](https://images.zenhubusercontent.com/58ac917f51aa2c30577510ef/0213f6c9-ba15-439a-a867-c7d3624a48d9)

**Comment by kevincrafts** [comment]
I was not able to replicate this behavior.

![image](https://user-images.githubusercontent.com/349713/92150557-cf04ae80-eddc-11ea-95db-54e718d4c122.png)

**Comment by jwfuller** [comment]
I think I figured out how to replicate:
- Select a term in the collection grid and save
- Go to the taxonomy for Collection Type and delete the term
- Reload the rendered version of the grid => error

**Comment by kevincrafts** [comment]
Rather than add a bunch of complicated validation to prevent the deletion of terms, we will just wrap this in a function to make sure the type value is set and degrade gracefully if it's not.

**Comment by cathysnider** [comment]
Here's the error I got when I deleted a Collection Type that was in use. In layman terms, the site went wacko.

<img width="580" alt="Screen Shot 2020-09-10 at 1 56 43 PM" src="https://user-images.githubusercontent.com/638275/92796529-7f8b2900-f36e-11ea-8cd0-6936506f4adb.png">
